source/systems/cloud_user_guide.rst

```diff
@@ -117,6 +118,292 @@ This includes data stored in a CSP public, free to access repositories, like
+and obtaining :ref:`rsa-token`.
+
+You must also be assigned to a Cloud project account.  To join a Cloud project,
+first request the project name from your PI, TL, or Portfolio Manager. Then use
+the AIM tool to `request access to that project
+<https://docs.rdhpcs.noaa.gov/accounts/accounts_and_projects.html#request-access-to-rdhpcs-projects>`_.
+
+Using ACTIVATE
+--------------
+
+See the :ref:`Workflow` diagram for an overview of the process.
+
+Users access the ACTIVATE platform through the `Parallel Works NOAA Portal`_,
+using the RSA Token authentication method.  On the landing page, enter your
+NOAA user name, and your PIN and SecurID OTP.
+
+
+`Foundational Parallel Works Training
+<https://drive.google.com/file/d/1Has2qJG6QZsaT3KTKp2VYBKBH4_6hrTO/view?ts=63f3b396>`_
+provides an introduction to features and function.
+An archive of `Parallel Works training sessions
+<https://sites.google.com/d/1QJ-MHpl1y0IEtzQUnIbjF2hUmMNQUMAo/p/1G8V0Mua9Dy7oUJ_wI36NAd3kMuMcHyGM/edit>`_ is also available.
+
+Storage Types and Storage Costs
+-------------------------------
+
+Three types of storage are available on a cluster.
+
+- Lustre: object storage for backup and restore and output files
+- Bucket/blob storage: a container for objects.
+- Contrib file system: a project’s custom software library.
+
+.. note::
+
+  An "object" is a file and any metadata that describes that file.
+
+Lustre file system
+^^^^^^^^^^^^^^^^^^
+
+Lustre is a parallel file system, available as ephemeral and persistent storage
+on the AWS, Azure, and GCP cloud platforms. A lustre file system can be
+attached and mounted on a cluster, and is accessible only from an active
+cluster. To create a lustre file system, access the Storage tab, and click Add
+Storage. You can create any number of lustre file systems. See `this article
+<https://parallelworks.com/docs/storage/creating-storage>`_ for
+information on creating a storage link.
+
+Bucket/Block blob storage
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Bucket storage and Block blob storage are containers for objects.
+An object is a file and any metadata that describes that file.
+Metadata can include use cases, such as
+data lakes, websites, mobile applications, backup and restore, archive,
+enterprise applications, IoT devices, or big data analytics. On AWS and GCP,
+the storage is called S3 bucket, and bucket respectively, whereas in Azure, the
+storage used is Block blob storage, which functions as a bucket and an NFS
+storage. Pricing information is available at this `link <https://aws.amazon.com/s3/pricing/>`_ .
+Projects using AWS, and GCP platforms can create as many buckets as needed, and
+mount them on a cluster. The project’s default bucket is accessible from the
+public domain using the keys.
+
+Contrib file system
+^^^^^^^^^^^^^^^^^^^
+
+The Contrib file system concept is similar to on-premise contrib. It is used to
+store files for team collaboration. You can use this storage to install custom
+libraries or user scripts.
+
```

Comment:
Add a paragraph here:  The contrib filesystem is built on the cloud provider's native NFS service, EFS in AWS, Azure Files in Azure, and GFS in GCP.